### DISCUSSION TITLE
emacs magit style Git workflow

### DISCUSSION THREAD

**Discussion by DoppleDankster** [zed-industries/zed#8941] [Feature Requests]
One of the coolest package on emacs is [magit](https://magit.vc/) a git tool that is so fantastic that i stick to emacs almost exclusively because of it.

I Know that the Zed team is working git integration within the editor, what about porting magit to rust and embed it directly in the editor ? 

The alternative would obviously be to use a plugin and i'm almost tempted to get into rust for that purpose alone

**Comment by failable** [comment]
Alternatives: https://github.com/altsem/gitu

But I personally use Zed + magit (in Emacs) now...

**Reply by DoppleDankster** [reply]
i kinda want integration in the editor directly wihtout having to switch to my term.
im also using zed and Magit in emacs and it's tiring

**Comment by ckgresla** [comment]
I too would absolutely love to see a magit like interface in zed!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links

**Comment by kriku** [comment]
Why it's became outdated?
